app/models/google_calendar/google_event.rb

```diff
@@ -0,0 +1,18 @@
+module GoogleCalendar
+  class GoogleEvent
+    extend Listing
+    extend Adding
+    extend Deleting
+
```

Comment:
How about changing Listing, Adding and Deleting into classes, some method could be private and you wouldn't have to pass many arguments through methods but keep them as class variable (at leas some).

Btw. it's more readable if you use keyword arguments https://robots.thoughtbot.com/ruby-2-keyword-arguments